src/model/EditorModel.ts

```diff
@@ -0,0 +1,17 @@
+import {PrimaryEditorRenderEngine} from "../logic/render/PrimaryEditorRenderEngine";
+import {BaseRenderEngine} from "../logic/render/BaseRenderEngine";
+import {IRect} from "../interfaces/IRect";
+import {IPoint} from "../interfaces/IPoint";
+
+export class EditorModel {
```

Comment:
wszystkie pola są statyczne (przez co to jest de facto singleton). to nie jest problemem jako taki, ale wiesz jakie są tego konsekwencje :P o ile w testach to sobie ładnie ogarniesz i nie będziesz tworzył różnych instancji modelu, to spoko